frontend/src/components/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Icon, Menu } from '@equinor/eds-core-react'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { config } from 'config'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { styled } from 'styled-components'
+import { Icons } from 'utils/icons'
+
+const StyledButton = styled(Button)`
+    width: 100px;
+    border-radius: 4px;
+`
+
+export const NavigationMenu = () => {
+    const { TranslateText } = useLanguageContext()
+    const [isOpen, setIsOpen] = useState(false)
+    const [anchorEl, setAnchorEl] = useState(null)
+    const openMenu = () => {
+        setIsOpen(true)
+    }
+    const closeMenu = () => {
+        setIsOpen(false)
+    }
+
+    const paths = [
+        { path: 'missionControl', label: 'Mission control' },
+        { path: 'history', label: 'Mission history' },
+        { path: 'inspectionOverview', label: 'Deck overview' },
```

Comment:
We call it inspection overview on large screen and deck overview on phone. Should use same naming